spectral_cube/tests/test_dask.py

```diff
@@ -58,3 +74,39 @@ def test_rechunk(data_adv):
     # note last element is 2 because the chunk size we asked for
     # is larger than cube - this is fine and deliberate in this test
     assert cube_new._data.chunksize == (1, 2, 2)
+
+
+def test_statistics(data_adv):
+    cube = DaskSpectralCube.read(data_adv)
+    stats = cube.statistics()
```

Comment:
@astrofrog I can reproduce (locally) the `IndexError` I commented on by forcing multiple chunks here:
```
    cube = cube.rechunk(chunks=(1, 3, 2))
```